### PR TITLE
[Chinese] Translate android_extension.md into chinese

### PR DESCRIPTION
**Pull request by chuan9**

### CODE REVIEW

**Comment by chuan9** [comment]
@jondong  @lisa0314 Please have a review

File: public/documentation/android/_data.json [hunk 1]
```diff
@@ -38,6 +38,9 @@
   "android_extensions": {
     "title": "Android extensions"
   },
+  "android_extensions_zh": {
+    "title": "安卓插件程序"
```
Comment (jondong) [inline public/documentation/android/_data.json]: 我们是不是统一翻译成扩展程序比较好？翻译成插件容易和Cordova的plugin混淆

File: public/documentation/android/android_extensions_zh.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# 安卓插件程序
```
Comment (jondong) [inline public/documentation/android/android_extensions_zh.md]: ditto

File: public/documentation/android/android_extensions_zh.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+本教程叙述了如何使用Java编写Crosswalk安卓插件程序。
```
Comment (jondong) [inline public/documentation/android/android_extensions_zh.md]: ditto

File: public/documentation/android/android_extensions_zh.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+<ol>
+
+<li>
+<p><strong>一个Crosswalk插件程序</strong></p>
+
+<p>插件程序包括：</p>
+
+<ul>
+<li>Java代码：打包到jar文件中的标准Android/Java类。</li>
+<li>JavaScript包装器：一个JavaScript文件，作用是暴漏Java代码到运行在Crosswalk上面的app中。</li>
```
Comment (jondong) [inline public/documentation/android/android_extensions_zh.md]: 暴露

File: public/documentation/android/android_extensions_zh.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+<p>在web程序中，Crosswalk插件程序被上面提到的Javascript包装器代码调用。在这个例子中，从插件程序返回的字符串将在DOM元素中被渲染。</p>
+
+</li>
+
+</ol>
+
+<p>这个工程也包含一些文件。这些文件的作用是帮助打包上面的文件到<code>.apk</code>文件，apk文件是安装在安卓机器上的。</p>
+
+本教程中所有的源码可以从[下载crosswalk-samples](https://github.com/crosswalk-project/crosswalk-samples/releases)获取, or或者在github https://github.com/crosswalk-project/crosswalk-samples (在`extensions-android`目录里)。
```
Comment (jondong) [inline public/documentation/android/android_extensions_zh.md]: 可以从[下载...](...)中获取
多余的or
或者从github<url>中获取(...)。

**Comment by jondong** [comment]
Thanks for the PR @chuan9 . It seems we'd better use ‘扩展’ instead of ‘插件’ and do a global replacement.

**Comment by chuan9** [comment]
@jondong Done, thanks for review.

**Comment by jondong** [comment]
LGTM, @baleboy @bspencer would you please help to merge? thanks

**Pull request merged**